src/components/elements/off-canvas-menu.tsx

```diff
@@ -172,9 +172,14 @@ export const OffCanvasMenuWrapper = (props: OffCanvasMenuWrapperProps) => {
                     >
                         {!props.is_ppc && (
                             <>
-                                <Text color="grey-5" mb="8px" size="14px">
+                                <Typography
+                                    element_color="grey-5"
+                                    mb="8px"
+                                    size="14px"
+                                    weight="normal"
```

Comment:
Instead of passing **weight="normal"** everywhere can't we just change it in the parent component?